Java_Class/LadderGame/src/domain/LadderMaker.java

```diff
@@ -0,0 +1,28 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LadderMaker {
+
+    private final int WIDTH;
+    private final int HEIGHT;
```

Comment:
인스턴스 변수의 네이밍 규칙에 맞지 않는군요. 

```suggestion
    private final int width;
    private final int height;

Reply:
아핫 감사합니다! 
상수로 쓰인다고 생각해서 저렇게 지었는데, 
이 경우에는 처음부터 값이 지정된 상수가 아니라 나중에 값이 할당되는 "변수"이기 때문에 일반 변수처럼 네이밍 해주면 된다고 이해하면 될까요??

Reply:
상수로 쓰인다면 `private final` 이면 안될 것 같은데요? 그리고 말씀하신 것처럼 객체가 생성될때마다 새로운 값이 할당되는, 명확하게 **변수** 입니다.